### PR TITLE
added parallax to background 

### PR DESCRIPTION
**Pull request by njosieb**
Fixes card #6 @cvharris

### CODE REVIEW

**Review by cvharris — CHANGES_REQUESTED** [review]
Let's wait until #13 merges before we merge this one.

**Comment by njosieb** [comment]
I merged conflicts with the master. @cvharris

File: src/_styles/shared/layout.less [hunk 1]
```diff
@@ -68,16 +68,17 @@ main {
...
+  height: 300%;
+  width: 200%;
   top: 0;
   left: 0;
   z-index: -1;
   clip: rect(auto, auto, auto, auto);
 
   // Fixed-position background image
   > .parallax-image {
+    background-image: url(denver-kitchen-remodel.jpg);
```
Comment (cvharris) [inline src/_styles/shared/layout.less]: If another element were to have the class `.parallax-image` it'll also have to get this `background-image`. We should only apply this specific `background-image` to an id target of some sort, possibly `#top-splash`

**Review by cvharris — CHANGES_REQUESTED** [review]

**Comment by njosieb** [comment]
Fixed parallax problems on the background image. @cvharris

**Comment by njosieb** [comment]
Made sure background image was in the right place. @cvharris

**Review by cvharris — APPROVED** [review]
Looks good! Great job! 👍 🥇 💯

**Pull request merged**